lib/jira/core.rb

```diff
@@ -16,21 +16,21 @@ def setup
       # @return [String] JIRA project endpoint
       #
       def url
-        @url ||= self.read(self.url_path)
+        @url ||=  ENV['JIRA_URL'] ||= self.read(self.url_path)
```

Comment:
With this syntax, double memoization is occuring - one with `@url` and another time with `ENV['..']`. We don't want to modify the `ENV`, so remove the second `=`s
eg. `@url ||= ENV['JIRA_URL'] || self.read(self.url_path)`